src/store/state.ts

```diff
@@ -112,9 +107,10 @@ export const emptyState: VQBState = {
   stepFormInitialValue: undefined,
   stepFormDefaults: undefined,
   selectedStepIndex: -1,
-  pipeline: [],
-  currentPipelineName: undefined,
-  pipelines: {},
+  currentPipelineName: 'my_pipeline',
+  pipelines: {
+    my_pipeline: [],
+  },
```

Comment:
I kind of dislike the idea of an hardcoded `my_pipeline` initial value. I don't like the `my_` part but this is admittedly a matter of taste :)
Wouldn't it be better to have `currentPipelineName` and `pipelines` set to `undefined` at first? And maybe handle those cases in the getters. After all, that's why there are getters, don't you think?

Reply:
That's a good point ! I gave a bit of thought into this:

I started as you said by having an `undefined` state for both `currentPipelineName` and `pipelines`, and therefore the current `pipeline`. This caused quite a massive amount of ts errors, indicating the case where pipeline is not defined should be handled.
I thought of implementing these, before realizing that may not be the scope of this PR:
- before, we had a `pipeline: []`, so no case where the pipeline is not defined
- we also do not have a UI case where no pipeline is selected, so it feels weird implementing one before knowing what to display in this case.
My conclusion: this should be also be a design issue and be another refactoring outside this PR.

I choosed the `my_pipeline` in accordance with what we choose for Toucan Toco's default dataset name `my_dataset`. I also didn't liked it at first, but it makes clear that it is a customizable name, that acts as a placeholder. The other possibilities that came to my mind ('default`, `defaultPipeline`, `pipeline`) didn't seem as clear to me. If you have any better suggestion, I'll be very pleased to change that!

Reply:
resolving in favor of https://github.com/ToucanToco/weaverbird/issues/441